.eslintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/*
+**/dist/*
+.github/actions/**/index.js"
```

Comment:
@blazejkustra I think we need to add `*.d.ts` as well. I'm testing with a `.d.ts` file and ESLint is complaining.

Reply:
Discussed it with Fabio on slack, this happened because tsconfig isn't configured for the catalog that Fabio created, resolved 👍